src/test/java/com/google/api/codegen/testdata/go_main_library.baseline

```diff
@@ -144,6 +163,16 @@ func (c *Client) SetGoogleClientInfo(name, version string) {
     }
 }
 
+// SetCallOptions sets the client's call options.
+func (c *Client) SetCallOptions(opts *CallOptions) {
+    c.callOptions = opts
```

Comment:
What do you think about making CallOptions exported so we wouldn't need these two functions?